docs/contents/releases/1-18/4/release-announcement.txt

```diff
@@ -0,0 +1 @@
+Amazon EKS Distro v1.18-4 is now available. This release includes upgraded versions of Kubernetes (from v1.18.9 to v1.18.16) and of Amazon Linus 2, which contains the latest security fixes. Amazon EKS Distro v1.18-4 builds of Kubernetes are available through ECR Public Gallery (https://gallery.ecr.aws/?searchTerm=EKS+Distro) and GitHub (https://github.com/aws/eks-distro)
```

Comment:
linux not linus

Reply:
Well that would've been not great. Good catch